api/comments/views.py

```diff
@@ -79,6 +79,7 @@ class CommentDetail(JSONAPIBaseView, generics.RetrieveUpdateDestroyAPIView, Comm
         is_abuse       boolean            has this comment been reported by the current user?
         has_children   boolean            does this comment have replies?
         can_edit       boolean            can the current user edit this comment?
+        new_mentions   list               guids of mentioned users in content of comment
```

Comment:
Looks like this is directly reflecting what is in the database.

![screen shot 2016-05-02 at 11 20 10 am](https://cloud.githubusercontent.com/assets/134455/14958269/e517ef3c-1057-11e6-88f9-2b294867d83a.png)